plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoTableHandle.java

```diff
@@ -47,12 +49,14 @@ public MongoTableHandle(
             @JsonProperty("remoteTableName") RemoteTableName remoteTableName,
             @JsonProperty("filter") Optional<String> filter,
             @JsonProperty("constraint") TupleDomain<ColumnHandle> constraint,
+            @JsonProperty("projectedColumns") Optional<List<MongoColumnHandle>> projectedColumns,
```

Comment:
Can we change to `Set<MongoColumnHandle>` like #14467? It will simplify some code, e.g. `MongoMetadata.containSameElements` can be deleted.